src/data/team.json

```diff
@@ -4,45 +4,45 @@
    "class_name": "selected",
    "officers": [{
          "name": "Jerry Qu",
-         "team": "Academic",
+         "team": "Administrative",
```

Comment:
any reason why we're calling it "Administrative"? EBoard might be better potentially

Reply:
i didn't know what to call it so i just chose administrative lol

should i change the team name to EBoard or Executive Board?

Reply:
let me ask

Reply:
"Leadership"